apps/backend/src/users/user.model.ts

```diff
@@ -50,6 +50,10 @@ export class User extends Model {
   @Column(DataType.STRING)
   encryptedPassword!: string;
 
+  @AllowNull(true)
+  @Column(DataType.STRING)
+  apikey!: string;
```

Comment:
@Bialogs API keys are currently stored unencrypted in the database. Will this be sufficient or is there wording in the SRG that recommends against this?

If the data is encrypted then it will not be able to simply be hashed, unless we only show the user their API key on generation and tell them to make note since it is not recoverable.

Reply:
@camdenmoors The actual API keys will need to be updated to only be returned to the frontend once, and then hashed using bcrypt just as a password is.